bears/vcs/git/GitCommitBear.py

```diff
@@ -124,6 +132,9 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
                 yield Result(self, 'HEAD commit has no message.')
             return
 
+        yield from self.check_auth_email(
+            auth_email[0][:-1], # Taking the email and removing \n
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/bears/vcs/git/GitCommitBear.py
+++ b/bears/vcs/git/GitCommitBear.py
@@ -133,7 +133,7 @@
             return
 
         yield from self.check_auth_email(
-            auth_email[0][:-1], # Taking the email and removing \n
+            auth_email[0][:-1],  # Taking the email and removing \n
             **self.get_auth_email_checks_metadata().filter_parameters(kwargs))
         yield from self.check_shortlog(
             shortlog,
```